src/Marketplace/MarketplaceScreen.js

```diff
@@ -32,11 +46,51 @@ const styles = StyleSheet.create({
     alignItems: 'flex-end',
     alignSelf: 'flex-end',
   },
+  produceCardsContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    flexWrap: 'wrap',
+    alignContent: 'center',
+    justifyContent: 'space-around',
+  },
+  produceCard: {
+    padding: 15,
+  },
 });
 
 export default function MarketplaceScreen({ navigation }) {
+  const [produceList, setProduceList] = useState([]);
+
+  const getProduce = () => {
+    const list = [];
+    base('Produce').select({
+      maxRecords: 20,
+    }).eachPage((records) => {
+      records.forEach((record) => {
+        list.push(record.fields);
+      });
+      setProduceList(list);
+    });
+  };
+
+  const produceCards = produceList.map((produce) => (
+    <ProduceCard
+      style={styles.produceCard}
+      key={produce.Name}
+      navigation={navigation}
+      image={carrotImage}
+      name={produce.Name}
+      price={produce.Price}
+      unit={produce.Unit}
+    />
+  ));
+
+  useEffect(() => {
+    getProduce();
```

Comment:
This looks good! How were you able to fix the issue of not loading upon mount?

Reply:
Unsure, I assume it's how I set up pushing the data to the useState variables